domains/noxe.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "DevNoxe",
+    "email": "vicktorelektrayt@gmail.com"
+  },
+  "record": {
+    "CNAME": "DevNoxe.github.io"
```

cannot access website, if the record is correct then please provide website link
```suggestion
    "CNAME": "devnoxe.github.io"
```